drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -8270,6 +8279,9 @@ void RenderingDeviceVulkan::compute_list_end(uint32_t p_post_barrier) {
 
 	memdelete(compute_list);
 	compute_list = nullptr;
+
+	// compute_list is no longer active
+	_THREAD_SAFE_UNLOCK_
 }
```

Comment:
On the 'dangerous' remark, in theory (possibly not in practice) this unlock could end up unlocking the lock we obtained on allocating our drawlist. There is nothing protecting this.